### DISCUSSION TITLE
Clone physical bodies

### DISCUSSION THREAD

**Discussion by JNightRider** [dyn4j/dyn4j#291] [Q&A]
Hello everybody...

I have recently been trying to make copies of physical bodies, however I discovered that the `Fixtures` list is a constant (`final`), so I cannot modify them (except their contents).

* Is there a way to clone these bodies? 
* Is it possible to consider supporting cloning (copying) of these bodies?

It would be great to clone (copy) them!

**Comment by wnbittle** [comment]
I like the idea of a allowing objects to be cloned - I'll create an enhancement issue for it.

**Comment by wnbittle** [comment]
Related issue has been created:  https://github.com/dyn4j/dyn4j/issues/293

**Reply by JNightRider** [reply]
Great, thanks for supporting this idea... I would like to help with this topic.
